app/facades/movie_facade.rb

```diff
@@ -7,6 +7,7 @@ def self.get_10_cast(api_id)
     MovieService.get_cast(api_id)[0..9]
```

Comment:
This works, I wonder if this is preferable to modifying the results in the model.

Reply:
Probably not actually - we should see if this would go better elsewhere , I would imagine model level